src/fontra/client/core/fontra-menus.js

```diff
@@ -0,0 +1,195 @@
+import * as html from "./html-utils.js";
+import { translate } from "/core/localization.js";
+import { MenuBar } from "/web-components/menu-bar.js";
+
+const mapMenuItemKeyToFunction = {
+  // File: getFileMenuItems, // TODO: this does not work, becuase viewController.fontController?.backendInfo is undefined. And I don't know why.
+  Font: getFontMenuItems,
+  Edit: getEditMenuItems,
+  View: getViewMenuItems,
+  Glyph: getGlyphMenuItems,
+};
+
+export function makeFontraMenuBar(menuItemKeys, viewController) {
+  const menuBarArray = [getFontraMenu()]; // Fontra-Menu at the beginning.
+
+  for (const itemKey of menuItemKeys) {
```

Comment:
Independent of the problem with the fontController you are seeing: you should *not* precompute the menu items. The should be constructed when getItems() is *called*.

Reply:
That was exactly the hint I needed. Thanks. It solves the other issue as well. I will push something in a bit.

Reply:
Ok, it's fixed now.